masterfabric-mobile/swift-camp
# CL - LazyVGrid Viper Implementation

## 📋 PR Description
This PR adds the implementation of a LazyVGridView following the VIPER architecture. A new button has been added to the HomeView, enabling navigation to the LazyVGridView screen. In the LazyVGridView items are created and loaded only when necessary.


---
## ✅ Checklist


---

## 🛠 Steps to Test

1. Launch the application.
2. Click the "Go to LazyVGrid" button
3. Ensure that LazyVGridView screen  appears
5. Scroll down - up to see new rectangles uploading while scrolling
6. Click the back button to verify that it navigates correctly to the HomeView.

---

## 🔗 Related Links

- Documentation: https://developer.apple.com/documentation/swiftui/lazyvgrid

---

### 📷 Screenshots (Optional)

![image](https://github.com/user-attachments/assets/8fe5f87a-4208-4c2a-a78c-494a08dccecd)![image](https://github.com/user-attachments/assets/7e3ffaaa-547e-4777-9445-75e13d56ed68)
